src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,57 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+)
+
+// Example: oci-factory upload -y --release tracks=1.0-22.04,risks=edge,eol=2025-06-01
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The integer-typed revision number of the image release"`
```

Comment:
[Nit]: I dont have enough context about the flag but I think `integer-typed` is not necessary here, probably `revision number of the image release` is enough together with the long help for the command if there needs to be any additional consideration.

Reply:
Makes sense!